mobile/lib/main.dart

```diff
@@ -180,10 +171,10 @@ class ImmichAppState extends ConsumerState<ImmichApp>
   @override
   initState() {
     super.initState();
-    initApp().then((_) => debugPrint("App Init Completed"));
+    unawaited(initApp().then((_) => debugPrint("App Init Completed")));
```

Comment:
What does `unawaited` do?

Reply:
It is merely a wrapper around the future to let the compiler know that we are deliberately ignoring the result of the future. I added them to remove the linting error from dcm's `avoid-async-call-in-sync-function` rule. However, on hindsight, we have a lot of such usage across our code-base and leaving them be as such seems harmless. So I've removed the rule now 😅